contrib/terraform/vsphere/variables.tf

```diff
@@ -23,7 +23,9 @@ variable "vsphere_datastore" {}
 
 variable "vsphere_user" {}
 
-variable "vsphere_password" {}
+variable "vsphere_password" {
+  sensitive = true
+}
```

Comment:
This change might be unrelated to this pull request I guess because this pull request is for gcp, not vsphere.